mautic/mautic
# Fix order of fields of form

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
In same cases with more complex forms and  conditional fields, reorder does not work.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. I don't know exactly list of fields to setup to reproduce issue, but you can try save and re-sort your forms and see If works like before
3. Of course code review should be good




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10801"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
